recipes/bison/all/conanfile.py

```diff
@@ -143,6 +159,7 @@ def package_info(self):
         self.output.info('Appending PATH environment variable: {}'.format(bindir))
         self.env_info.PATH.append(bindir)
 
+        return
```

Comment:
is it an intended change ?

Reply:
Oh, no. I added it because I wanted to try the `--enable-relocate` option (which does not work satisfactory).
But I got all tests running on Widows and Linux with this line added.

Reply:
I looked at it again and it's ok to remove these:
- `BISON_PKGDATADIR` is not required because my relocate patch will use a relative path to the bison executable. It is still possible to override `BISON_PKGDATADIR` (as it is used in the sources, but it is not required anymore)
- the `BISON_INSTALLER_ROOT` is not required anymore as the installer is dead